app/src/main/java/com/anysoftkeyboard/gesturetyping/GestureTypingDetector.java

```diff
@@ -23,9 +24,11 @@
 public class GestureTypingDetector {
     private static final String TAG = "GestureTypingDetector";
 
-    private static final double CURVATURE_THRESHOLD = Math.toRadians(160);
-    // How many points away from the current point to we use when calculating curvature?
-    private final int mCurvatureSize;
+    private static final double CURVATURE_THRESHOLD = Math.toRadians(170);
+    // How many points away from the current point do we use when calculating curvature?
+    private static final int CURVATURE_NEIGHBORHOOD = 1;
+    // How far away do two points of the gesture have to be?
+    private final int mMinPointDistance;
```

Comment:
This variable is only used with its square value, do you think we can just store the square value?

Reply:
Sure, I changed it so we store the square.